codegen/generate.pl

```diff
@@ -1,8 +1,18 @@
-#!/usr/bin/perl
```

Comment:
You can keep the shebang here and not mess with the license

Reply:
Good catch, that was an accident